modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -582,10 +586,12 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
       assert(txRequest.transactions || txRequest.unsignedTxs, 'Unable to find transactions in txRequest');
       const unsignedTx =
         txRequest.apiVersion === 'full' ? txRequest.transactions![0].unsignedTx : txRequest.unsignedTxs[0];
-      txToSign = unsignedTx.signableHex;
+      txOrMessageToSign = unsignedTx.signableHex;
       derivationPath = unsignedTx.derivationPath;
     } else if (requestType === RequestType.message) {
-      throw new Error('MPCv2 message signing not supported yet.');
+      assert(txRequest.messages, 'Unable to find messages in the txRequest');
```

Comment:
should assert length as well
```suggestion
      assert(txRequest.messages?.length, 'Unable to find messages in the txRequest');
```